.infrastructure/helm-chart/todoapp/templates/pvc.yml

```diff
@@ -0,0 +1,13 @@
+
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc-data
+  namespace: {{ .Values.todoapp.namespace }}
+spec:
+  volumeName: pv-data
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 1Gi
```

Comment:
Try to use this value from values.yml and Fix EoF

Reply:
fixed in [fix EoF, add serviceAccount YAML file, use pv storage value from value.yml](https://github.com/mate-academy/devops_todolist_kubernetes_task_12_helm_charts/pull/30/commits/72ec2793e112758cfac44d2779c6d423be281e55) commit